test/integration/no_kubernetes_test.go

```diff
@@ -45,6 +45,7 @@ func TestNoKubernetes(t *testing.T) {
 			name      string
 			validator validateFunc
 		}{
+			{"ErrorStartNoK8sWithVersion", errorStartNoK8sWithVersion},
```

Comment:
dont we need to add a new Step to Delete the cluster ? so the Next subtest "start" can actually start without K8s?

Reply:
forget this one, maybe we dont need. since the First one wont actually create a Config ? and second "start" is still Fresh?

Reply:
errorStartNoK8sWithVersion() should not successfully create a cluster, so we won't have to delete.

Yes, errorStartNoK8sWithVersion() won't create a config, and will allow for a fresh start.